README.md

```diff
@@ -1,5 +1,32 @@
 # Constitution
 
-The governing rules of Hackers at Cambridge.
+The governing rules of "Hackers at Cambridge", hereafter (and in the rest of the repository) referred to as the "Society".
 
-[Read the constitution here](constitution.md).
+[Read the constitution here](constitution.md). You can also find a web version
+[on our website](https://hackersatcambridge.com/constitution).
+
+# This Repository
+
+As laid out in the [constitution](constitution.md#society-repository), this repository is used to list all rules and guidelines of the Society that are not defined or contained in the constitution. This repository exists for the benefit of the executive committee, but is free to read for the general public for those who may be interested.
+
+# Our Website
+
+The official website of the Society is located at [https://hackersatcambridge.com](https://hackersatcambridge.com).
+
+# Making changes to this repository
+
+Any changes to this repository must require a majority vote at a meeting of the Executive Committee. Please view [the template](.github/PULL_REQUEST_TEMPLATE.md) for more information.
+
+Individual files in the repository may relax these requirements for certain files and folders, but these requirements are *only* relaxed for changes which *only* files and folders with relaxed restrictions.
```

Comment:
I don't understand what this means

Reply:
I always write that when tired

Gonna give it another go -_-